lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@jcoyne , @mjgiarlo I tried it but it does not work!
I removed the 3 lines from engine.rb and added require 'legato' in engine.rb
and removed require 'legato' from analytics.rb. Getting the same error:
undefined method `sufia__download' for #<Legato::Management::Profile:0x007f85f68b12e8>